migration/db-id/mssql/migration.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```

Comment:
This file should not be in the mssql directory. it should be one level up

Reply:
ok